modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java

```diff
@@ -293,6 +294,42 @@ public void testGetObjectDefinitionObjectFieldsPageWithPagination()
 		assertContains(objectField3, (List<ObjectField>)page3.getItems());
 	}
 
+	@Override
+	@Test
+	public void testGetObjectField() throws Exception {
+		super.testGetObjectField();
+
+		// Set unique value
+
+		ObjectField objectField = randomObjectField();
+
+		objectField.setObjectFieldSettings(
+			new ObjectFieldSetting[] {
+				new ObjectFieldSetting() {
+					{
+						name = "uniqueValues";
```

Comment:
There is a constant for this String (ObjectFieldSettingConstants.NAME_UNIQUE_VALUES)